var/spack/repos/builtin/packages/perl-xml-parser/package.py

```diff
@@ -35,3 +35,13 @@ class PerlXmlParser(PerlPackage):
     version('2.44', 'af4813fe3952362451201ced6fbce379')
 
     depends_on('expat')
+
+    def configure_args(self):
+        args = []
+
+        el = join_path(self.spec['expat'].prefix, 'lib')
+        ei = join_path(self.spec['expat'].prefix, 'include')
```

Comment:
See https://github.com/LLNL/spack/blob/develop/lib/spack/spack/util/prefix.py for a list of supported attributes. That's where all the magic happens.